Makefile

```diff
@@ -1,11 +1,11 @@
 
 BUILD_DIR=build
-COVERAGE_DIR=${BUILD_DIR}/coverage
+COVERAGE_DIR=$(BUILD_DIR)/coverage
```

Comment:
I saw that about half used `{ }` and half used `( )`, and did a quick `:%s/\${\([^}]\+\)}/$(\1)/g` on it. I'll revert it to just the necessary change.